openlibrary/macros/SearchNavigation.html

```diff
@@ -0,0 +1,30 @@
+$def with ()
+
+$ q = query_param("q")
+
+<ul class="work-menu">
```

Comment:
```suggestion
<ul class="nav-bar">
```
We initialize the book page sticky title on pages where `.work-menu` is found.  As such, errors are appearing in the JS console for these search results pages.

The `nav-bar` class was added in #6665 to circumvent this problem.